### PR TITLE
implemented service meta parameters

### LINKED ISSUE #435: Support for service/node "meta" parameters
Docs:
https://www.consul.io/docs/agent/services.html
https://www.consul.io/docs/agent/options.html#_node_meta

Error message:
```
Evaluation Error: Error while evaluating a Resource Statement, Consul::Service[foo]: has no parameter named 'meta'
```

### PR DESCRIPTION
**Pull request by sc0rp10**
Fixed #435

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -43,6 +46,7 @@
   $service_name        = $title,
   $tags                = [],
   $token               = undef,
+  $meta                = {},
```
Comment (bastelfreak) [inline manifests/service.pp]: Can you please add a datatype to the new parameter? We would like to have them for all new parameters.

**Comment by bastelfreak** [comment]
Hi @sc0rp10, a few of our needed gems don't work properly within our specified versions. I will implement a fix in https://github.com/solarkennedy/puppet-consul/pull/437. Please rebase after that got merged (or rebase against my featurebranch as soon as it is green).

**Comment by sc0rp10** [comment]
@bastelfreak i'll wait for green build of #437, ok?

**Comment by bastelfreak** [comment]
@sc0rp10 yes sounds good.

**Comment by jardleex** [comment]
Hey folks,
any progress on this one? Just found #441 and this one here.
We would be very happy to find this in the module.
Thank you!

**Comment by bastelfreak** [comment]
@jardleex I cannot merge this, because there are conflicts. Are you interested in rebasing it and provide the code as a new PR?

**Comment by jardleex** [comment]
I think this can get closed due to #466

**Pull request closed without merging**